homeassistant/components/keyboard_remote/__init__.py

```diff
@@ -236,7 +238,13 @@ async def async_keyrepeat(self, path, name, code, delay, repeat):
             while True:
                 self.hass.bus.async_fire(
                     KEYBOARD_REMOTE_COMMAND_RECEIVED,
-                    {KEY_CODE: code, DEVICE_DESCRIPTOR: path, DEVICE_NAME: name},
+                    {
+                        KEY_CODE: code,
+                        VALUE: 2,
+                        TYPE: KEY_VALUE_NAME.get(2),
```

Comment:
Make `KEY_UP`, `KEY_DOWN`, `KEY_HOLD` constants:
```suggestion
                        TYPE: `KEY_HOLD`,
```

Reply:
It doesn't really matter, but we should at least not use `dict.get()` here.

The backticks were not intentional, sorry about that.

Reply:
I'll change it to using the string constant `"key_hold"` as `"key_up"` and `"key_down"` are already used as constants elsewhere in the code.